### ISSUE TITLE
Test cases not added

### ISSUE THREAD

**Issue by Maha-Shawky** [asmaahamdy25/chart-task#1]
Please use any testing framework (cypress, testcafe, .....) to write the two test cases, so the` npm test` command pass both successfully

**Comment by asmaahamdy25** [comment]
There is file name test case .txt

في الاثنين، ١ فبراير ٢٠٢١ ٢:٣٢ م Maha-Shawky <notifications@github.com> كتب:

**Comment by Maha-Shawky** [comment]
Yes, but it's text file describing what tests should do, you need to write the  unit test cases itself
Look at App.test.js this is a test file example

**Comment by Maha-Shawky** [comment]
If you run `npm test` test runner will try to test the test cases written there

**Comment by asmaahamdy25** [comment]
Ok , I don't understand that

في الاثنين، ١ فبراير ٢٠٢١ ٢:٤٦ م Maha-Shawky <notifications@github.com> كتب:

**Comment by asmaahamdy25** [comment]
Hi Maha
I try to write test cases for the task busun8082I try to write npm test to
check how it Work I face  err٢٠٢I can't fix .I don't have experience in
test cases so
I'm sorry for your time. I will improve my self in test

Thanks
Asmaa
في الاثنين، ١ فبراير ٢٠٢١ ٢:٥٢ م Asmaa Hamdy AbdulRahman <sun80826@gmail.com>
كتب:
